pkg/scheduler/internal/cache/cache.go

```diff
@@ -70,7 +70,7 @@ type cacheImpl struct {
 	headNode *nodeInfoListItem
 	nodeTree *nodeTree
 	// A map from image name to its imageState.
```

Comment:
A map from image name to its `ImageStateSummary`.

Reply:
yeah, imageState need change to be ImageStateSummary